domains/mano.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "infofbnr"
+    },

```

Comment:
Please add either your email or if you dont want to give out your email at least one social media account so that we can contact you.